src/metabase/api/dashboard.clj

```diff
@@ -791,7 +784,9 @@
     GET /api/dashboard/1/params/abc/values?def=100"
   [id param-key :as {:keys [query-params]}]
   (let [dashboard (api/read-check Dashboard id)]
-    (param-values dashboard param-key query-params)))
+    ;; If a user can read the dashboard, then they can lookup the chain-filter. This also works with sandboxing.
```

Comment:
nit: technically these endpoints are used for all filters, not just chain filters, I believe